bitnami/redis-cluster/7.4/debian-12/prebuildfs/opt/bitnami/scripts/liblog.sh

```diff
@@ -49,7 +58,14 @@ log() {
 #   None
 #########################
 info() {
-    log "${GREEN}INFO ${RESET} ==> ${*}"
+    local msg_color="$GREEN"
+    local reset_color="$RESET"
+    local color_check="${BITNAMI_COLOR:-true}"
+    if ! [[ "$color_check" =~ ^(yes|true)$ ]]; then
```

Comment:
Update if condition format, and please implement the logic the other way around: if true then set colors